EpiAware/src/epimodel.jl

```diff
@@ -60,24 +60,34 @@ struct DirectInfections <: AbstractEpiModel
     data::EpiData
 end
 
-function (epi_model::DirectInfections)(recent_incidence, unc_I_t)
-    nothing, epi_model.data.transformation(unc_I_t)
+function (epimodel::DirectInfections)(_It, latent_process_aux)
+    epimodel.data.transformation.(_It)
 end
 
 struct ExpGrowthRate <: AbstractEpiModel
     data::EpiData
 end
 
-function (epi_model::ExpGrowthRate)(unc_recent_incidence, rt)
-    new_unc_recent_incidence = unc_recent_incidence + rt
-    new_unc_recent_incidence, epi_model.data.transformation(new_unc_recent_incidence)
+function (epimodel::ExpGrowthRate)(rt, latent_process_aux)
+    latent_process_aux.init .+ cumsum(rt) .|> exp
 end
 
 struct Renewal <: AbstractEpiModel
     data::EpiData
 end
 
-function (epi_model::Renewal)(recent_incidence, Rt)
-    new_incidence = Rt * dot(recent_incidence, epi_model.data.gen_int)
-    [new_incidence; recent_incidence[1:(epi_model.data.len_gen_int-1)]], new_incidence
+function (epimodel::Renewal)(_Rt, latent_process_aux)
+    I₀ = epimodel.data.transformation(latent_process_aux.init)
+    Rt = epimodel.data.transformation.(_Rt)
+
+    r_approx = fast_R_to_r_approx(Rt[1], epimodel)
```

Comment:
we don't need to call this fast and it isn't informative. I suggest `R_to_r` and then it can have methods later on or a name that explicitly touches on the method being being used.

Reply:
The reason for `fast_` is that other options were eigenvalue or more black box root finding.

Reply:
But I'll defer to your wisdom here!

Reply:
I've changed the name.

Reply:
my point is more that we should be able to dispatch to different methods from the same function right? vs naming the function and then having to change that name

Reply:
also "wisdom" - love it